.asf.yaml

```diff
@@ -30,6 +30,12 @@ github:
     rebase:  false
   protected_branches:
     master:
+      required_status_checks:
+        strict: true
+        contexts:
+          - license
+          - pecl
+          - required
```

Comment:
Ping @kezhenxu94 to review the name. I am struggling with these names every time.